labcodes/confetti-ds
# Dialog message justify

**Link to task:**
https://labcodes.atlassian.net/browse/CSYS-296

**Staging app:** https://dialog-message-justify--confetti-storybook.netlify.app/

**How to test:**

1. Open [Confetti storybook](https://dialog-message-justify--confetti-storybook.netlify.app/)
2. Open the `Dialog` -> `Standard` component
3. Open the dialog
4. Verify that the close button icon is centralized inside the button

**Description of your solution:**
The `.lab-dialog__close-button--message` class was added to the `Standard` dialog close button.
